org-brain.el

```diff
@@ -266,6 +274,42 @@ Ignores \"dotfiles\"."
     (directory-files-recursively
      org-brain-path (format "^[^.].*\\.%s$" org-brain-files-extension))))
 
+(defun org-brain-replace-links-with-visible-parts (raw-str)
+  "Return `str', with all bracketed links replaced with the link
```

Comment:
Should probably be "Return RAW-STR, with all....". Argument to Emacs functions should be written in ALL CAPS in the docstring.